src/frontend/SettingsDialog.ui

```diff
@@ -1,178 +1,133 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ui version="4.0">
```

Comment:
![settings_cutoff](https://user-images.githubusercontent.com/9459891/43120519-4b2f2762-8f3c-11e8-839a-14548a82ed77.png)
The text is being cut off.